scarlet2/interpolation.py

```diff
@@ -286,6 +306,8 @@ def resample_ops(kimage, shape_in, shape_out, res_in, res_out, interpolant=Quint
         interpolant
         )
     
+    # TODO: Apply flip on rotation if any
+    # k_resampled *= flip_sign.prod()
```

Comment:
If I'm not completely mistaken, you need the `flip_sign` at some place. Otherwise, please remove from method signature.